backend/corpora/common/corpora_orm.py

```diff
@@ -267,7 +271,7 @@ class DbDeploymentDirectory(Base):
     created_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
     updated_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
 
-    dataset = relationship("DbDataset")
+    dataset = relationship("DbDataset", back_populates="deployment_directories")
 
 
 class DbContributor(Base):
```

Comment:
Missing a relationship here to DbDatasetContributor

Reply:
is that relationship needed here